test/raw.html

```diff
@@ -25,11 +25,20 @@
 
   <script>
 
-    window.localStorage.setItem('core-localstorage-test', 'hello world');
-    var storage = document.querySelector('#localstorage');
+    var storage;
     
     suite('basic', function() {
 
+      before( function() {
```

Comment:
Conventionally, we prefer Mocha's `tdd` interface. The equivalents of `before` and `after` are `setup` and `teardown`, respectively.